web/app.php

```diff
@@ -3,24 +3,15 @@
 use Symfony\Component\HttpFoundation\Request;
 umask(0002);
 
-/**
- * @var Composer\Autoload\ClassLoader
- */
-$loader = require __DIR__.'/../app/autoload.php';
-include_once __DIR__.'/../var/bootstrap.php.cache';
-
-// Enable APC for autoloading to improve performance.
-// You should change the ApcClassLoader first argument to a unique prefix
-// in order to prevent cache key conflicts with other applications
-// also using APC.
-/*
-$apcLoader = new Symfony\Component\ClassLoader\ApcClassLoader(sha1(__FILE__), $loader);
-$loader->unregister();
-$apcLoader->register(true);
-*/
+require __DIR__.'/../vendor/autoload.php';
+if (PHP_VERSION_ID < 70000) {
```

Comment:
This file is a part of symfony :), you can consult with this here https://github.com/symfony/symfony-standard/blob/v3.3.2/web/app.php#L7